packages/now-cli/src/commands/dns/rm.ts

```diff
@@ -19,15 +19,17 @@ export default async function rm(
   args: string[],
   output: Output
 ) {
-  const { authConfig: { token }, config } = ctx;
+  const {
+    authConfig: { token },
+    config,
+  } = ctx;
   const { currentTeam } = config;
   const { apiUrl } = ctx;
   const debug = opts['--debug'];
   const client = new Client({ apiUrl, token, currentTeam, debug });
-  let contextName = null;
 
   try {
-    ({ contextName } = await getScope(client));
+    await getScope(client);
```

Comment:
Do we still need to call getScope() even though the result was removed?

Reply:
@styfle Yes because if the user is not logged in it should throw an error https://github.com/zeit/now/blob/704446397a94a5ce4685fa76fed35366268a52a2/packages/now-cli/src/commands/dns/rm.ts#L34:L37

Reply:
I see. I was thinking that the first API call to getDNSRecordById() would handle it but I guess we don't want to throw or else we'll be getting errors in sentry about users who are logged out. Seems good then, thanks 👍